conf/upgrade.yaml

```
@@ -56,9 +56,12 @@ UPGRADE:
   UPGRADE_WITH_HTTP_PROXY: false
   # Default size of client's which we use for content host upgrade.
   CLIENTS_COUNT: "8"
-  # SATQE_SHARED_PASSWORD for remote execution
+  # Satellite's REMOTE_SSH_PASSWORD
+  REMOTE_SSH_PASSWORD:
```

Keep this password/key values commented, as you'll be passing this as dynaconf env var's
```suggestion
  #REMOTE_SSH_PASSWORD:
```

---

It is required otherwise GitHub action will get fail for code coverage.